modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/pages/questions/Question.es.js

```diff
@@ -190,6 +197,22 @@ export default withRouter(
 			[markAsAnswerMessageBoardMessage, answers.items, fetchMessages]
 		);
 
+		useEffect(() => {
+			const body = document.body;
+			const html = document.documentElement;
+
+			const docHeight = Math.max(
+				body.scrollHeight,
+				body.offsetHeight,
+				html.clientHeight,
+				html.scrollHeight,
+				html.offsetHeight
+			);
+
+			const winHeight = window.innerHeight;
+			setIsPageScroll(docHeight > winHeight);
+		}, [question, answers]);
+
```

Comment:
@Lucas2691 I think this is too much, for scroll to an specific element, this could be done this way:

```
<a href="#google"></a>

<div id="google"></div>
```

I got this example from: 

https://stackoverflow.com/questions/24739126/scroll-to-a-specific-element-using-html